vms/platformvm/network/gossip_test.go

```diff
@@ -29,7 +32,17 @@ func TestGossipMempoolAddVerificationError(t *testing.T) {
 		TxID: txID,
 	}
 
-	mempool, err := pmempool.New("", prometheus.NewRegistry(), nil)
+	mempool, err := pmempool.New(
+		&config.Internal{
+			UpgradeConfig: upgrade.Config{
+				EtnaTime: time.UnixMilli(1),
```

Comment:
A bunch of tests broke on this PR because some of them use old txs that aren't supported post-etna - so all existing tests are configured to use the pre-etna fork.